src/app/core/facades/checkout.facade.ts

```diff
@@ -85,6 +85,7 @@ export class CheckoutFacade {
   basketChange$ = this.basketChangeInternal$.asObservable();
   basketError$ = this.store.pipe(select(getBasketError));
   basketInfo$ = this.store.pipe(select(getBasketInfo));
+  basketInfoError$ = this.basketInfo$.pipe(map(info => (info?.length ? info[0].error : undefined)));
```

Comment:
Maybe this should be its own selector?

Also, maybe this is the wrong place to ask this so please let me know if that's the case but can you explain the store structure here a bit? @SGrueber 
![image](https://user-images.githubusercontent.com/34165455/171160209-5e64a3d1-32e4-481f-b804-e58dd4a08d87.png)

How come there's both a `basket.info` as well as a `info` substate in the basket state?

Reply:
basket.info contains info of the get basket (or the result of any call that returns the whole basket ). The info sub state contains the info of the last basket operation, e.g. addProductToCart. Because after each basket operation a get basket is triggered we need both info values.